Resources/Prototypes/Traits/disabilities.yml

```diff
@@ -65,7 +65,6 @@
 - type: trait
   id: Uncloneable
   category: Physical
-  points: 4
```

Comment:
Because it would still be "Free points" even at 1. The downside straight up doesn't come up in our codebase that often because we still have things like CPR, "Free" Space/Epipens(You can literally start with 2), a psionic power that revives you, paramedics that still have portable crew monitors, etc.

Reply:
I think it should be 1